packit_service/worker/handlers/copr.py

```diff
@@ -85,13 +86,14 @@ def __init__(
         package_config: PackageConfig,
         job_config: JobConfig,
         event: dict,
+        celery_task: Optional[CeleryTask] = None,
     ):
         super().__init__(
             package_config=package_config,
             job_config=job_config,
             event=event,
         )
-
+        self.celery_task = celery_task
```

Comment:
Since 2 handlers already contain this initialization, what about creating another class doing this? Something like

```python
class RetriableJobHandler(JobHandler):
    def __init__(
        self,
        package_config: PackageConfig,
        job_config: JobConfig,
        event: dict,
        celery_task: Optional[CeleryTask] = None,
    ):
    super().__init__(
            package_config=package_config,
            job_config=job_config,
            event=event,
       )
       self.celery_task = celery_task
```

(sorry for the formatting, github editor makes it hard to manipulate)

This way, it's more obvious which handlers actually retry